hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/builder/GridComponentBuilder.java

```diff
@@ -120,6 +121,22 @@ public static Button buildLink(final String idSuffix, final String idPrefix, fin
         return addColumn(i18n, grid, E::getName, "header.name", columnId, 100D);
     }
 
+    /**
+     * Add controllerId column to grid
+     *
+     * @param grid
+     *            to add the column to
+     * @param i18n
+     *            message source for internationalization
+     * @param columnId
+     *            column ID
+     * @return the created column
+     */
+    public static Column<ProxyTarget, String> addControllerIdColumn(final Grid<ProxyTarget> grid,
+            final VaadinMessageSource i18n, final String columnId) {
+        return addColumn(i18n, grid, ProxyTarget::getControllerId, "header.controllerId", columnId, 100D);
```

Comment:
Could you please define the constant for minWidth (100D) parameter not to duplicate it for each method, thanks!

Reply:
Good idea. Done ✅